File: 1 Semestre/Analise e Projeto de Algoritmos (Mauricio)/Aula 6 Estruturas de Controle (Repeticao)/Exercicio 5.py
'''
5) Elabore um algoritmo que leia um numero inteiro
e o inverta. Veja alguns exemplos:

Num = 1234 -> Invertido = 4321
Num = 564378 -> Invertido = 873465
'''



num = int(input("Digite o valor do numero: "))
div = num
alg = 0
i = 0
inv = 0

while div > 0:
    div //= 10
    i += 1

while num > 0:
    alg = num % 10
    num //= 10
    inv += alg * (10 ** i)
    i -= 1

print(inv//10)

File: 2 Semestre/Sistemas Embarcados (C)/Aula 4/Exercicio 5.c
#include <stdio.h>

int main(){
    int a,b,r;

    printf("Digite o valor A: ");
    scanf("%i",&a);

    printf("Digite o valor B: ");
    scanf("%i",&b);

    r = (a*a) + (b*b);

    printf("O resultado de %i^2 + %i^2 e %i",a,b,r);}